docs/server/project_setup.rst

```diff
@@ -195,6 +195,19 @@ of the Pootle server:
     (env) $ pootle update_stores --project=my-project
 
 
+.. note:: Alternatively you can do this with the following set of Pootle FS
+   commands:
```

Comment:
This shouldn't be alternatively, this should be the default way.  So I'd remove all `update_stores` found earlir and replace it with what you have here.